File: src/main/java/hacker/solution/ArrayListDemo.java
package hacker.solution;

import java.math.BigDecimal;
import java.util.*;

public class ArrayListDemo {
    public static void main(String []args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        ArrayList[] rows = new ArrayList[n];
        for (int i=0; i<n; i++) {
            int d = sc.nextInt();
            ArrayList<Integer> row = new ArrayList<>();
            for (int j = 0; j < d; j++) {
                row.add(sc.nextInt());
            }
            rows[i]=row;
        }
        int q = sc.nextInt();
        for (int i=0; i<q; i++) {
            int x = sc.nextInt();
            int y = sc.nextInt();
            try {
                System.out.println(rows[(x - 1)].get(y - 1));
            } catch (IndexOutOfBoundsException e) {
                System.out.println("ERROR!");
            }
        }
        sc.close();
    }
}

File: src/main/kotlin/hacker/solutionk/ArrayListDemo.kt
package hacker.solutionk

import java.lang.NullPointerException
import java.util.*

object ArrayListDemo {
    @JvmStatic
    fun main(args: Array<String>) {
        val sc = Scanner(System.`in`)
        val n = sc.nextInt()
        val rows = arrayOfNulls<ArrayList<Integer>>(n)
        for (i in 0 until n) {
            val d = sc.nextInt()
            val row = ArrayList<Integer>()
            for (j in 0 until d) {
                row.add(Integer(sc.nextInt()))
            }
            rows[i] = row
        }
        val q = sc.nextInt();
        for (k in 0 until q) {
            val x = sc.nextInt()
            val y = sc.nextInt()
            try {
                println(rows[x-1]?.get(y-1))
            } catch (e: IndexOutOfBoundsException) {
                println("ERROR!")
            } catch (e1: NullPointerException) {
                println("ERROR!")
            }
        }
        sc.close()
    }
}